src/timelines.js

```diff
@@ -141,6 +141,7 @@ export default Kapsule({
         state.transDuration = val?700:0;
       }
     },
+    segmentTooltipHtml: {},
```

Comment:
Changing this should not trigger a component update. Please add `triggerUpdate: false`.